### PR TITLE
Add step by step local sync description

### PR DESCRIPTION
**Pull request by christophschlosser**
Step by step instructions for using N1 with a local sync instance.

This could make setting up a self hosted instance easier.

### CODE REVIEW

File: docs/N1andLocal.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    accounts: [
+```
+* Visit http://localhost:5555/accounts
+* Copy the output between the {}
+* Paste it in the config file, after the stuff from 8.
+* Paste after the stuff from 11. The accountTokens have to be there for every account.
+```
+]
+    accountTokens:
+        "account_id value without qutation marks": "account_id value with qutation marks"
```
Comment (dillonforrest) [inline docs/N1andLocal.md]: I spotted a small typo! `qutations` => `quotations`, in both the key and the val. I'm totally nitpicking here hehe.

**Comment by dillonforrest** [comment]
Hey @christophschlosser this looks great so far! Thanks for putting in the time to do this! I'm gonna run through the steps here myself later today to make sure we're not missing anything. Meanwhile, would it be cool if you made a quick change to fix some typos that I pointed out?

File: docs/N1andLocal.md (hunk 1 above)
Comment (christophschlosser) [inline docs/N1andLocal.md]: I hope you are ;)
I'm used to a much rougher code review.

**Comment by christophschlosser** [comment]
hey @dillonforrest I updated my patch with the fixed typo and a few other things.
If there is still stuff missing feel free to criticize it again and I'll update it.
Maybe this needs a tutorial with screenshots some time later.

**Comment by bengotow** [comment]
Hey @christophschlosser — thanks for these improvements. We've collected a bunch of feedback in the last couple days and we've compiled it all together in `CONTRIBUTING.md`. I've updated that page to include the details you added in `N1andlocal.md`. I think it'd be better to keep things together rather than adding another file, so I'm going to close this pull request. As of https://github.com/nylas/N1/commit/7000c586459837bca72110c6510d9b3b72684e2c, CONTRIBUTING.md should include your notes.

**Pull request closed without merging**